pkg/vet/api_resources_test.go

```diff
@@ -48,7 +48,7 @@ rolebindings                                   rbac.authorization.k8s.io      tr
 		},
 		{
 			name: "invalid line",
-			lines: `NAME                              SHORTNAMES   APIGROUP                       NAMESPACED   KIND
+			lines: `NAME                              SHORTNAMES   APIVERSION                       NAMESPACED   KIND
 namespaces                        ns                                          other        Namespace
 deployments                       deploy       apps                           true         Deployment
 clusterroles                                   rbac.authorization.k8s.io      false        ClusterRole
```

Comment:
This should have version information, e.g. `rbac.authorization.k8s.io/v1`. It'd also be good to update the test so that we catch issues like this earlier b/299119028#comment13. Might be enough to just check the header line of `kubectl api-resources` to see if it still produces what `nomos` expects.

Reply:
the test case has been updated to include version information to resemble the current api-resource output. The unit test has covered a scenario that checks for the header line, I added another scenario that will closely resemble the APIGROUP/APIVERSION switch